ivy/functional/backends/torch/general.py

```diff
@@ -297,10 +297,13 @@ def inplace_update(
         if keep_input_dtype:
             val = ivy.astype(val, x.dtype)
         (x_native, val_native), _ = ivy.args_to_native(x, val)
+        x_native.data = val_native
```

Comment:
You don't have to add this line, the lines below basically in place updates the native array